assets/src/edit-story/app/media/utils/useFFmpeg.js

```diff
@@ -235,6 +235,53 @@ function useFFmpeg() {
     [getFFmpegInstance]
   );
 
+  /**
+   * Transcode and mute a video using FFmpeg.
+   *
+   * @param {File} file Original video file object.
+   * @return {Promise<File>} Transcoded video file object.
+   */
+  const transcodeMuteVideo = useCallback(
+    async (file) => {
+      //eslint-disable-next-line @wordpress/no-unused-vars-before-return
+      const trackTiming = getTimeTracker('load_mute_video_transcoding');
+
+      try {
+        const ffmpeg = await getFFmpegInstance(file);
+
+        const tempFileName = uuidv4() + '-muted.' + MEDIA_TRANSCODED_FILE_TYPE;
+        const outputFileName =
+          getFileName(file) + '-muted.' + MEDIA_TRANSCODED_FILE_TYPE;
+
+        await ffmpeg.run(
+          // Input filename.
+          '-i',
+          file.name,
+          ...FFMPEG_SHARED_CONFIG,
```

Comment:
```suggestion
          '-vcodec',
          'copy',
```

This way only the audio is removed and no other optimization or transcoding is performed.

This will make the process much faster.